hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JetService.java

```diff
@@ -208,38 +205,30 @@ public void handle(Packet packet) throws IOException {
         networking.handle(packet);
     }
 
+    @Override
+    public void memberAdded(MembershipServiceEvent event) {
+        // nop
+    }
 
-    private class CancelJobsMigrationListener implements MigrationListener {
-
-        @Override
-        public void migrationStarted(MigrationEvent migrationEvent) {
-            Set<Address> addresses = nodeEngine.getClusterService().getMembers().stream()
-                                               .map(Member::getAddress)
-                                               .collect(Collectors.toSet());
-            // complete the processors, whose caller is dead, with TopologyChangedException
-            liveOperationRegistry.liveOperations
-                    .entrySet().stream()
-                    .filter(e -> !addresses.contains(e.getKey()))
-                    .flatMap(e -> e.getValue().values().stream())
-                    .forEach(op ->
-                            Optional.ofNullable(executionContexts.get(op.getExecutionId()))
-                                    .map(ExecutionContext::getExecutionCompletionStage)
-                                    .ifPresent(stage -> stage.whenComplete((aVoid, throwable) ->
-                                            completeExecution(op.getExecutionId(),
-                                                    new TopologyChangedException("Topology has been changed")))));
-            // send exception result to all operations
-            liveOperationRegistry.liveOperations
-                    .values().stream().map(Map::values).flatMap(Collection::stream)
-                    .forEach(op -> op.completeExceptionally(new TopologyChangedException("Topology has been changed")));
-        }
-
-        @Override
-        public void migrationCompleted(MigrationEvent migrationEvent) {
-        }
-
-        @Override
-        public void migrationFailed(MigrationEvent migrationEvent) {
-        }
+    @Override
+    public void memberRemoved(MembershipServiceEvent event) {
```

Comment:
This event is a local event and is guaranteed to be fired after a member is removed.